### DISCUSSION TITLE
Item level auto-approval

### DISCUSSION THREAD

**Discussion by miguelverweij** [microsoft/PowerPlatform-Catalog#17] [Ideas]
The options for General and Publisher level auto-approval are great. The flexibility of the product can be significantly improved by allowing auto-approval on single Catalog items. This way a team can be allowed to update their department's template, but not add anything else. Not sure what the impact would be on the solution though.

**Comment by MattB-msft** [comment]
I covered this in the response to #16 you created,  

Are you asking for independence between Publisher and Catalog Item?  IE allow a given Catalog Item to be auto published but not general things form the publisher?
